pds4indextools/pds4_create_xml_index.py

```diff
@@ -222,40 +185,123 @@ def process_schema_location(file_path):
     return xsd_urls
 
 
-def process_tags(xml_results, key, root, namespaces, prefixes, args):
+def process_tags(label_results, key, root, namespaces, prefixes):
     """Process XML tags based on the provided options.
 
     If the --xpaths command is used, the XPath is converted into a format that
     contains the names and namespaces of all the parent elements of that element.
     If the --xpaths command is not used, the XPath is converted into the
     associated element tag of that element, and given its associated namespace. These
-    values then replace their old versions in the xml_results dictionary.
+    values then replace their old versions in the label_results dictionary.
 
 
     Inputs:
-        xml_results    A dictionary containing XML data to be processed.
-        key            The key representing the XML tag to be processed.
-        root           The root element of the XML tree.
-        namespaces     A dictionary containing XML namespace mappings.
-        prefixes       A dictionary containing XML namespace prefixes.
-        args           Command-line arguments.
+        label_results    A dictionary containing XML data to be processed.
+        key              The key representing the XML tag to be processed.
+        root             The root element of the XML tree.
+        namespaces       A dictionary containing XML namespace mappings.
+        prefixes         A dictionary containing XML namespace prefixes.
     """
-    if args.xpaths:
-        key_new = convert_header_to_xpath(root, key, namespaces)
-        for namespace in prefixes.keys():
-            if namespace in key_new:
-                key_new = key_new.replace(
-                    '{'+namespace+'}', prefixes[namespace]+':')
-        xml_results[key_new] = xml_results[key]
-        del xml_results[key]
-    else:
-        key_new = convert_header_to_tag(key, root, namespaces)
-        for namespace in prefixes.keys():
-            if namespace in key_new:
-                key_new = key_new.replace(
-                    '{'+namespace+'}', prefixes[namespace]+':')
-        xml_results[key_new] = xml_results[key]
-        del xml_results[key]
+    key_new = convert_header_to_xpath(root, key, namespaces)
+    for namespace in prefixes.keys():
+        if namespace in key_new:
+            key_new = key_new.replace(
+                '{'+namespace+'}', prefixes[namespace]+':')
+    label_results[key_new] = label_results.pop(key)
+
+
+def renumber_xpaths(xpaths):
```

Comment:
Why did you take out all the comments I wrote, and the additional explanatory text in the docstring? I think they were all very useful for someone looking at this code in the future and trying to figure out what it does.